src/com/pentalog/pentastagiu/homework2_bookCandyShop/MembershipStatus.java

```diff
@@ -0,0 +1,25 @@
+package com.pentalog.pentastagiu.homework2_bookCandyShop;
+
+public enum MembershipStatus {
+    GOLD(0.20),
```

Comment:
f bine ca ai legat membership status de valoarea discountului, astfel ai un singur loc in care e implementata legatura dintre ele si daca va fi nevoie sa faci o schimbare va f usor